public/newsitem.js

```diff
@@ -0,0 +1,32 @@
+function newsItem(apiResponseElement,articleID) {
+  this.headline = apiResponseElement.headline;
+  this.bodyText = apiResponseElement.bodyText;
+  this.shortUrl = apiResponseElement.shortUrl;
+  this.thumbnail = apiResponseElement.thumbnail;
+  this.id = articleID
+}
+
+newsItem.prototype.getHeadlineHTML= function() {
+  var headline = document.createElement('a')
+  headline.href = this.shortUrl;
+  headline.classList.add('headline');
+  headline.id = `Article #${this.id} headline`;
```

Comment:
this will result in three ids, Article, 4, and headline. Article and headline are then not unique. It should probably be something like:
```
headline.id = `Article-#${this.id}-headline`;
```
If that's what you're going for?